src/components/Switch/Switch.stories.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { inlineTemplate } from '../../../storybook/decorators/storyTemplates';
+import { Switch } from './index';
+import { boolean, select, withKnobs } from '@storybook/addon-knobs';
+import { DOCUMENTATION_URL } from '../../../storybook/constants';
+
+const stories = storiesOf('Widgets', module);
+stories.addDecorator(withKnobs);
+
+stories.addWithInfo('Switch', '', () => {
+  const bsSize = select('Size', Switch.SWITCH_BS_SIZES, 'normal');
+  const onColor = select('On Color', Switch.SWITCH_COLORS, 'primary');
+  const offColor = select('Off Collor', Switch.SWITCH_COLORS, 'default');
+
+  const props = { bsSize, title: bsSize, id: 'bsSize-example' };
+  if (onColor) props.onColor = onColor;
+  if (offColor) props.offColor = offColor;
+
+  let story = (
+    <div>
+      <Switch
+        {...props}
+        onChange={action('onChange')}
+        animate={boolean('animate', true)}
+        disabled={boolean('disabled', false)}
+        readonly={boolean('readonly', false)}
+        inverse={boolean('inverse', false)}
+        tristate={boolean('tristate', false)}
+      />
+    </div>
+  );
+  return inlineTemplate({
+    title: 'Switch',
+    documentationLink:
+      DOCUMENTATION_URL.PATTERNFLY_ORG_WIDGETS + '#bootstrap-switch',
```

Comment:
Maybe a string literal.

Reply:
Going to leave this as is, its only in this one place and consistent with other components.